website_sale_product_detail_attribute_image/views/templates.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2019 Tecnativa - Sergio Teruel
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <template id="product_detail_attribute_image" inherit_id="website_sale.product" active="True" customize_show="True" name="Attribute's Images">
+        <xpath expr="//span[@itemprop='image']/.." position="inside">
+            <div class="row mt16 mb0 product-detail-attributes">
+                <t t-foreach="image_attributes" t-as="image_attribute">
+                    <div class="col-md-2">
+                        <div class="text-center"> <span t-field="image_attribute.attribute_id.name"/> </div>
+                        <div class="text-center">
+                            <img class="img"
+                                 t-attf-src="/web/image/product.attribute/#{image_attribute.attribute_id.id}/website_product_detail_image"/>
+                        </div>
+                        <t t-foreach="image_attribute.value_ids" t-as="value">
+                            <div class="text-center"> <span t-field="value.name"/> </div>
```

Comment:
The same about showing these backend string here. You are not mentioning this in the README, nor the label of the attribute.